codecarbon/core/measure.py

```diff
@@ -68,8 +68,23 @@ def do_measure(self) -> None:
                 self._ram_power = power
                 logger.info(
                     f"Energy consumed for RAM : {self._total_ram_energy.kWh:.6f} kWh"
-                    + f". RAM Power : {self._ram_power.W} W"
+                    + f".RAM Power : {self._ram_power.W} W"
```

Comment:
Why removing this space after the point ? Maybe just move the point in the previous string ?